.golangci.yml

```diff
@@ -131,16 +131,5 @@ linters-settings:
       - name: var-naming
   testifylint:
     disable:
-      - float-compare
       - go-require
-    enable:
-      - bool-compare
-      - compares
-      - empty
-      - error-is-as
-      - error-nil
-      - expected-actual
-      - len
-      - require-error
-      - suite-dont-use-pkg
-      - suite-extra-assert-call
+    enable-all: true
```

Comment:
OK, so you don't have formatter I think, my point was to know how you used testifylint.

Here you reply by question.

I didn't check yet to see if golangci-lint support the option I was talking about.

Reply:
I don't see it here yet

https://golangci-lint.run/usage/linters/#testifylint

There might be aPR about this new setting.

But you could use testifylint directly and use the -fix, you know. This way you will have the last version and will be able to use the setting I'm talking about

Reply:
The changelog of golangci-lint is very clear about version.

Anyway, thanks for confirming you didn't use the fix mode, or the formatter option, because the f function would have been used when there is a message